test/blocks/bulk-publish-v2/bulk-publish-v2.test.js

```diff
@@ -0,0 +1,184 @@
+import { expect } from '@esm-bundle/chai';
+import { readFile, sendKeys, sendMouse } from '@web/test-runner-commands';
+import { delay, waitForRemoval } from '../../helpers/waitfor.js';
+import { mockFetch, unmockFetch } from './mocks/fetch.js';
+import { setConfig } from '../../../libs/utils/utils.js';
+import './mocks/authentication.js';
+
+const conf = { miloLibs: 'http://localhost:2000/libs' };
+setConfig(conf);
+
+document.body.innerHTML = await readFile({ path: './mocks/body.html' });
+const { default: init } = await import('../../../libs/blocks/bulk-publish-v2/bulk-publish-v2.js');
+
+const testPage = 'https://main--milo--adobecom.hlx.page/drafts/sarchibeque/bulk-publish-test';
```

Comment:
Is this the right place for this to live on SharePoint? Feels like it fits better in `/tools/`, but that's just my opinion

Reply:
oh yeah good point, I will move it there